docs/run_sims.rst

```diff
@@ -110,3 +110,53 @@ When the simulation and postprocessing is all complete, run ``buildstock_aws
 --clean your_project_file.yml``. This will clean up all the AWS resources that
 were created on your behalf to run the simulations. Your results will still be
 on S3 and queryable in Athena.
+
+Google Cloud Platform
+~~~~~~~~~~~~~~~~~~~~~
+
+Running a batch on GCP is done by calling the ``buildstock_gcp`` command line
+tool.
+
+.. command-output:: buildstock_gcp --help
+   :ellipsis: 0,8
+
+GCP Specific Project configuration
+..................................
+
+TODO: add more GCP configuration details here
+
+For the project to run on GCP, you will need to add a section to your config
+file, something like this:
+
+.. code-block:: yaml
+
+    gcp:
+      job_identifier: national01
+      gcp_project: myorg_project
+      region: us-central1
+      artifact_registry: buildstockbatch
+      gcs:
+        bucket: mybucket
+        prefix: national01_run01
+      use_spot: true
+      batch_array_size: 10000
```

Comment:
To reflect [the current state of the schema](https://github.com/rewiringamerica/buildstockbatch/blob/gcp/buildstockbatch/schemas/v0.3.yaml):
- `gcp_project` → `project`
- no `gcs` nor `use_spot`

Reply:
Those changes are in the PR I'm about to send! Sorry they're out of order - I pulled these changes out of that branch.

(I'm using `gcp_project` to distinguish it from the BSB "project" that's being run.)